crypto/threads_none.c

```diff
@@ -121,4 +121,10 @@ int CRYPTO_atomic_add(int *val, int amount, int *ret, CRYPTO_RWLOCK *lock)
     return 1;
 }
 
+int openssl_init_fork_handlers(void)
+{
+    /* Is failing the right thing to do? */
```

Comment:
It's complicated.  Whenever possible, we want to get the safe-fork-behavior by default. which is why the flag is "NO_ATFORK"    But maybe non-pthreads should not fail.  Since it's just not there.